main.go

```diff
@@ -47,11 +48,11 @@ func initMageTools() {
 		}
 	}()
 
-	if mgtool.GetCWDPath("") != mgtool.GetGitRootPath("") {
+	if mgpath.FromCWD("") != mgpath.FromGitRoot("") {
```

Comment:
IMO would be nice to avoid a 3-letter abbreviation in the API here - how about `mgpath.FromWorkDir`?

Reply:
Yeah thats better, fixed :+1: